initcpio/hooks/oroot

```diff
@@ -39,7 +38,7 @@ run_hook() {
               mount -o \$(awk '{if (\$2 == \"/\") {print \$4}}' /etc/fstab) \"$root\" /run/oroot || mount \"$root\" /run/oroot;
               rsync -ax --no-whole-file --inplace --delete / /run/oroot --exclude boot --exclude dev --exclude mnt --exclude proc --exclude run --exclude sys --exclude tmp --exclude usr/bin/overlay_flush;
               umount /run/oroot &&
-              rm -r /run/oroot &&
+              rmdir /run/oroot &&
```

Comment:
I see you already fixed it https://github.com/bluerider/liveroot/pull/23/commits/7814d4d4269573ad317b7dfc17e1c48348f637e0